docs/source/executing-operations/subscription-support.mdx

```diff
@@ -146,30 +146,34 @@ subscription:
   mode:
     passthrough:
       all: # The router uses these subscription settings UNLESS overridden per-subgraph
-        path: /subscriptions # The URL path to use for subgraph subscription endpoints (Default: /ws)
+        path: /subscriptions # The absolute URL path to use for subgraph subscription endpoints (Default: /ws)
       subgraphs: # Overrides subscription settings for individual subgraphs
         reviews: # Overrides settings for the 'reviews' subgraph
-          path: /ws # Overrides '/subscriptions' defined above
-          protocol: graphql_transport_ws # The WebSocket-based subprotocol to use for subscription communication (Default: graphql_ws)
+          path: /ws # Absolute path that overrides '/subscriptions' defined above
+          protocol: graphql_transport_ws # The WebSocket-based subprotocol to use for subscription communication (Default: graphql_transport_ws)
 ```
 
 This example enables subscriptions in **passthrough mode**, which uses long-lived WebSocket connections.
 
 The router supports the following WebSocket subprotocols, specified via the `protocol` option:
 
-- `graphql_ws`
-  - Legacy subprotocol used by the [`subscriptions-transport-ws` library](https://github.com/apollographql/subscriptions-transport-ws), which is **unmaintained**
-  - For backward compatibility, this is the default value.
 - `graphql_transport_ws`
   - Used by the [graphql-ws](https://github.com/enisdenjo/graphql-ws) library
-  - This subprotocol is recommended for GraphQL server libraries implementing WebSocket-based subscriptions.
-
-> ⚠️ **Your router must use whichever subprotocol is expected by each of your subgraphs!**
+  - This subprotocol is the **default value** and is recommended for GraphQL server libraries implementing WebSocket-based subscriptions.
+- `graphql_ws`
+  - Legacy subprotocol used by the [`subscriptions-transport-ws` library](https://github.com/apollographql/subscriptions-transport-ws), which is **unmaintained** but provided for backward compatibility. 
```

Comment:
We need to flip the bullets here- `graphql_ws` is for `graphql-ws` library, and vice versa.

Reply:
ugh, I got tripped up after reading again https://github.com/enisdenjo/graphql-ws/blob/master/PROTOCOL.md#communication stating `The WebSocket sub-protocol for this specification is: graphql-transport-ws.`

Reply:
I know it's soooo confusing, I don't know what's the right answer here, maybe we could rename that field and use `protocol_name` or I don't know :p I'm open to any suggestions